<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>304</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>맑은 고딕</family>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>콤시스 센티넬원 리포트</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelConsole">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>콘솔 주소</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboConsole">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>361</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string>https://apne1-samsungsds.sentinelone.net</string>
    </property>
    <item>
     <property name="text">
      <string>https://apne1-samsungsds.sentinelone.net</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>https://apne1-1101-nfr.sentinelone.net</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>https://apne1-1001.sentinelone.net</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>https://apne1-1002.sentinelone.net</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelApitk">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>API 토큰</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConfirm">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>확인</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelGrp">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Group</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDate">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>기간</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateStart">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximumDate">
     <date>
      <year>2110</year>
      <month>12</month>
      <day>31</day>
     </date>
    </property>
    <property name="minimumDate">
     <date>
      <year>2010</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
    <property name="timeSpec">
     <enum>Qt::LocalTime</enum>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEnd">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximumDate">
     <date>
      <year>2110</year>
      <month>12</month>
      <day>31</day>
     </date>
    </property>
    <property name="minimumDate">
     <date>
      <year>2010</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
    <property name="date">
     <date>
      <year>2010</year>
      <month>1</month>
      <day>2</day>
     </date>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>~</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboGrp">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRepair">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>수정</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineApitk">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>IwdlMxqistO4F57BwUQ8Gwkkr4ATPPXXGWYtfbwbI1t5Gx9qiyKBQXsmc9JkSPopdXaH1XwOFawDyDRR</string>
    </property>
    <property name="placeholderText">
     <string>API Token 값을 발급받아 붙여넣기 하세요</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelAcc">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Account</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSite">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>142</y>
      <width>56</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Site</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboAcc">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboSite">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelAcc_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>110</y>
      <width>0</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Account 선택하세요</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSite_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>140</y>
      <width>0</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Site 선택하세요</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelGrp_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>170</y>
      <width>0</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Group 선택하세요</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReporting">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>240</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>보고서 생성</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnChk">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>200</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>결정</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDirectory">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="prbar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>501</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
